how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb

```diff
@@ -186,18 +217,22 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "OYyvqCycvgTG"
+   },
    "source": [
     "Try to change the values and re-execute to see your runs appear in the dashboard."
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "uqMDkxyKvgTG"
+   },
    "source": [
-    "All `basic-colab-example` runs are grouped in [this dashboard view](https://app.neptune.ai/o/common/org/colab-test-run/experiments?viewId=2eb46477-0064-4617-a182-17f8a2fcc06a).\n",
+    "All `basic-colab-example` runs are grouped in [this dashboard view](https://app.neptune.ai/o/common/org/quickstarts/experiments?split=tbl&dash=charts&viewId=96fe9215-15ff-4bf8-9096-9d06a99a8b36).\n",
     "\n",
-    "![colab-runs](https://neptune.ai/wp-content/uploads/docs-api-tour.png)\n",
+    "![colab-runs](https://user-images.githubusercontent.com/41324509/184083881-90568b31-166b-4fef-9887-8cec587de1ae.png)\n",
```

Comment:
All images should be on our WP. Please contact Partycja, and she will upload the file